bin/projects/dbatools/dbatools/Configuration/Config.cs

```diff
@@ -0,0 +1,67 @@
+﻿using System;
```

Comment:
Absolutely. That's a relic from the auto-conversion Visual Studio was using in conjunction with the limitations of the single-file library.
Was gonna clean that up once the proof of concept has been worked out. If you have automated tools for that, go ahead.